package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ifrau",
-  "version": "0.22.1",
+  "version": "1.0.0",
```

Comment:
I think this can be `0.23.0` actually -- semver has a weird thing for `0.x` releases where the second number is treated as "major".

Reply:
oh ... awesome! ... looks like i'll need to bump up to 0.24.0